content/_changelogs/9.6.0.md

```diff
@@ -0,0 +1,63 @@
+## 9.6.0
+
+_Released 4/25/2022_
+
+**Experiments:**
+
+- The Sessions beta feature has been enhanced by the new beta feature which
+  enables Multi-Origin support. To reflect the intent of using these experiments
+  together, the `experimentalSessionSupport` configuration option has been
+  replaced by the `experimentalSessionAndOrigin` configuration option. The
+  behaviors and usage of the `cy.session()` command has not been changed. Please
+  update your configuration to use the new `experimentalSessionAndOrigin` option
+  to continue to opt-in to this beta feature.
+
+**Features:**
+
+- We are excited to announce a new beta feature to enable Multi-Origin support.
```

Comment:
Beta and experimental are 2 very different things, we shouldn't use the world Beta in this case. https://docs.google.com/document/d/1NdZdTHW0y9WCPbK6gKNZgekFvxYG7CN7F3YK93R9wHg/edit

```suggestion
- We're excited to announce a new experimental feature to enable Multi-Origin support.
```